configuration/debian/tedge_agent/postinst

```diff
@@ -12,4 +12,16 @@ if ! getent passwd tedge-agent > /dev/null; then
     adduser --quiet --system --no-create-home --ingroup tedge-agent --shell /usr/sbin/nologin tedge-agent
 fi
 
+### Create file in /etc/sudoers.d directory
+echo "%tedge-agent   ALL = (ALL) /usr/bin/tedge_agent" > /etc/sudoers.d/tedge-agent
+
+if [ -f "/etc/sudoers.d/010_pi-nopasswd" ]; then
+    echo "%tedge-agent   ALL = (ALL) NOPASSWD: /usr/bin/tedge_agent" > /etc/sudoers.d/tedge-agent-nopasswd
+fi
+
+## Enable the sm services if the device is connected to c8y cloud
+if [ -f "/etc/tedge/mosquitto-conf/c8y-bridge.conf" ]; then 
+    systemctl start tedge-agent.service
+    systemctl start tedge-mapper-sm-c8y.service    
```

Comment:
I think enabling of the services is missing here. They should also be restarted after a system restart.